modules/gitops-installing-argocd-agent-using-helm.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assembly:
+//
+// * argocd-agent-installation/gitops-installing-argocd-agent-using-helm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argocd-agent-using-helm_{context}"]
+= Installing the Argo CD Agent by using a Helm chart
+
+Use the Helm chart provided by the Argo CD Agent project to install the Agent component on the workload cluster. Argo CD Agent supports two modes for installation, *managed* and *autonomous*.
+
+.Procedure
+
+. Install the Managed Agent.
+In *managed* mode, the Agent must communicate with the spoke Redis server. However, the default network policy created by the {gitops-title} Operator does not allow this. To enable communication, you must create a custom network policy.
```

Comment:
Question for Jayendra, is this only needed for managed mode, or do we need to do this for autonomous mode as well?

I would have thought it would be needed for both, but as written it only says for managed mode.